setup.py

```diff
@@ -28,7 +28,7 @@
     version=VERSION,
     description='A Command-Line Interface framework',
     long_description=README,
-    license='MIT',
+    license='MIT License',
```

Comment:
We set `license` to `MIT` according to Python's official doc:

https://packaging.python.org/en/latest/guides/distributing-packages-using-setuptools/#license

![image](https://user-images.githubusercontent.com/4003950/204953978-71a784de-fc3f-434e-b167-6d89bf216849.png)

```suggestion
    license='MIT',
```

Let's not change for now.